compiler/src/dotty/tools/dotc/sbt/ExtractDependencies.scala

```diff
@@ -199,15 +263,18 @@ private class ExtractDependenciesCollector(implicit val ctx: Context) extends tp
    *  because it refers to these classes or something defined in them.
    *  This is always a superset of `topLevelInheritanceDependencies` by definition.
    */
```

Comment:
Documentation needs to be updated since this is now a set of pair, same for the def just below.